components/event-preview/event-preview.vue

```diff
@@ -2,8 +2,8 @@
   <article
     :class="
       isIncoming
-        ? 'px-8 py-2 mb-10 border-solid border-8 border-green-500'
-        : 'px-8 py-2 mb-10 border-solid border-4 grey-border-200'
+        ? 'px-8 py-2 mb-10 border-solid border-8 border-mtl-green-500'
+        : 'px-8 py-2 mb-10 border-solid border-4 border-grey-border-200'
```

Comment:
Should we not have a `border-mtl-grey-200` instead of `border-grey-border-200`?

Reply:
Very good point! I also found more "gray" color issues while fixing this. Thanks!